projects/cashmere/src/lib/datepicker/datepicker.component.scss

```diff
@@ -0,0 +1,296 @@
+@import '../sass/colors';
+@import '../sass/typography';
+@import '../sass/functions';
+@import '../sass/mixins';
+
+.hc-datepicker-content {
+    width: 296px;
```

Comment:
Looks like our container isn't quite big enough for the content with padding (see attached).  If you increase width to 310 it should account for the 8px padding being applied.
<img width="341" alt="screen shot 2019-01-28 at 10 29 57 am" src="https://user-images.githubusercontent.com/22795893/51854349-d3571480-22e7-11e9-8694-d6398cdc8abc.png">

Reply:
If this change is made - it looks like the `.hc-datepicker-content .hc-calendar` is getting set inline somewhere, but that should also change to 310px so the right side of the header is aligned with the wider width.  This value might be calculated automatically, I just wasn't sure where.

Reply:
Thank you for pointing this out, It was a simple style sheet change